lib/MetricsConsumer.js

```diff
@@ -124,6 +128,9 @@ class MetricsConsumer {
             log.end();
             return done();
         }
+        if (data.site) {
+            data.site = this._removePreferredRead(data.site);
```

Comment:
I don't think this is the right place to do this processing, the metrics topic messages should not include preferred read info as it's not related. Let me figure where could be a better place.

Reply:
ok, thanks @jonathan-gramain